tasks/fix-cat2.yml

```diff
@@ -1,25 +1,27 @@
 ---
-- name: "MEDIUM | RHEL-07-010030 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
-  command: "true"
-  changed_when: no
-  when:
-      - rhel7stig_gui
-      - rhel_07_010030
-  tags:
-      - RHEL-07-010030
-      - dod_logon_banner
-      - notimplemented
+### RHEL-07-010030 | RHEL-07-010040 combined as related tasks in regards to a config file no other content will be in.
+- name: "MEDIUM | RHEL-07-010030 | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
+  block:
+      - name: "MEDIUM | RHEL-07-010030 | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
+        template:
+            src: 01-banner-message.j2
+            dest: /etc/dconf/db/local.d/01-banner-message
+            owner: root
+            group: root
+            mode: '0644'
 
-- name: "MEDIUM | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the approved Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
-  command: "true"
-  changed_when: no
+      - name: "MEDIUM | RHEL-07-010030 | RHEL-07-010040 | PATCH | The Red Hat Enterprise Linux operating system must display the Standard Mandatory DoD Notice and Consent Banner before granting local or remote access to the system via a graphical user logon."
+        command: dconf update
+        register: rhel_07_010040dconfout
+        changed_when: rhel_07_010040dconfout.rc <= 1
+        failed_when: rhel_07_010040dconfout.rc > 1
   when:
       - rhel7stig_gui
+      - rhel_07_010030
       - rhel_07_010040
   tags:
-      - RHEL-07-010040
+      - RHEL-07-010030
```

Comment:
This should have both STIG IDs as tags. i.e. add `RHEL-07-010040`

Reply:
committed.